### ISSUE TITLE
Error running site:layout

### ISSUE THREAD

**Issue by ThomasFinnern** [joomla-extensions/boilerplate#14]
After creating a base component [TestAjax](https://github.com/ThomasFinnern/Joomla3_TestAjax) from boilerplate the view had an error. I could fix it 

#### What is wrong:
Error message ![2018-08-05 19_10_28-test ajax](https://user-images.githubusercontent.com/7040580/43688147-d6e7ea5e-98e3-11e8-8acf-089bdde2c825.png)

Following code parts do not match and issue the error in the page.php

```
default.php:
$data = new stdClass;
$data->text = 'Hello Joomla!';
echo $layout->render($data);
```
```
page.php
defined('_JEXEC') or die;
extract($displayData);
echo $text;
```

#### Here is the solution. 
there a two possible was to fix it and i prefer (2) 

* 1
```
page.php
defined('_JEXEC') or die;
extract($displayData);

$text = $displayData->text;

echo $text;
```

* 2
```
default.php:

$data = array('text' => 'Hello Joomla! (2)');

echo $layout->render($data);
```

**Comment by roland-d** [comment]
@ThomasFinnern This was already fixed with commit b00e77f993f6484537a4121e1533fba7103ec14e Do you still have this issue?

**Comment by ThomasFinnern** [comment]
sorry, didn't follow it up as i could fix it with my examples.
Thanks for your work.
May i rember you to J!4 ... ;-)
